services/user.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+    username: str,
+    password: str,
+    first_name: str = None,
+    last_name: str = None,
+    email: str = None,
+) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        first_name=first_name if first_name else "",
+        last_name=last_name if last_name else "",
+        email=email if email else "",
+    )
```

Comment:
Consider handling potential errors when creating a user, such as when the username already exists. This will prevent runtime exceptions and improve user feedback.